x-pack/plugins/metrics_entities/server/modules/host_entities/host_entities.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "host_ent",
```

Comment:
As discussed, no, these are ok. Different solutions can write their own modules in the REST API and have their solution name in the modules. Those modules can then pull in these transforms if they want to or not.

So the REST API as seen below groups transforms into modules and you can have multiple modules which have the same transforms if they want to. This allows solution specific modules which can re-use some of the same transforms and/or create their own transforms for their own modules or for other people's modules.

Current grouping of modules as seen below is:
* "host_metrics"
* "host_entities"
* "network_metrics"
* "network_entities"
* "user_entities"
* "user_metrics"


But (as an example) if someone came along and decided they wanted to wire up their own custom module and use some of the same transforms such as this one, they are able to. They can also wire up new transforms instead of re-using these.

Basically, it's one layer so we don't have to duplicate/copy transforms, but rather we can share them across the concept of modules.